articles/quantum-InstallConfig.md

```diff
@@ -15,131 +15,28 @@ ms.topic: article
 # Installing the Quantum Development Kit
 
 The Quantum Development Kit can be used either with the Visual Studio 2017 integrated development environment, or with the command line and a development editor such as Visual Studio Code.
-We'll walk through both methods in this install guide.
+
+<table>
+<tr>
+<td>
+[<img src="media/install-guide-select-vs-2017.png" width="90%">](xref:microsoft.quantum.install.vs-2017)
+<div class="nextstepaction">
+[Install for Visual Studio 2017 on Windows 10](xref:microsoft.quantum.install.vs-2017)
+</div>
+</td>
+
+<td>
+[<img src="media/install-guide-select-vs-code.png" width="90%">](xref:microsoft.quantum.install.vs-code)
```

Comment:
On the same spirit, I would replace the screenshot with one that shows a command line on the front and VS-code on the back.

Reply:
If it's alright, I'd like to defer the screenshot, since that could make the call to action a bit busy graphically speaking.